lms/templates/seq_module.html

```diff
@@ -3,6 +3,16 @@
 
 <div id="sequence_${element_id}" class="sequence" data-id="${item_id}" data-position="${position}" data-ajax-url="${ajax_url}" data-next-url="${next_url}" data-prev-url="${prev_url}">
   <div class="path"></div>
+    % if masquerade_override:
```

Comment:
See comment above about renaming this variable.  By having the variable name be specific to the expected behavior in the template, the business logic remains on the server-side and the code in the template is easier to understand.  For example, a reader of the mako template could be confused of why masquerading is related to the displayed message.